analyzer/src/main/kotlin/MavenSupport.kt

```diff
@@ -475,6 +475,9 @@ class MavenSupport(workspaceReader: WorkspaceReader) {
                     Pair(type, url)
                 }
             } else {
+                if (connection.isNotEmpty()) {
+                    log.info { "Ignore Maven scm-connection URL '$connection' of unexpected format." }
```

Comment:
Nit: I'd say "Ignoring Maven SCM connection URL '$connection' of unexpected format.".

Reply:
Commit message nit: "Log" with upper-case "L".